include/eve/arch/cpu/tags.hpp

```diff
@@ -33,26 +34,39 @@ namespace eve
   struct aggregated_ : cpu_
   {
     using parent = cpu_;
+    static constexpr bool regular_logical_register = true;
   };
 
   //================================================================================================
-  // Concept for discriminatig aggregated from non-agrgegated ABI
+  // Concept for discriminatig aggregated from non-agregated ABI
+  //================================================================================================
   template<typename T> concept regular_abi = !std::same_as<T,aggregated_>;
 
   //================================================================================================
   // Dispatching tag for emulated SIMD implementation
   struct emulated_ : cpu_
   {
-    static constexpr std::size_t bits           = 128;
-    static constexpr std::size_t bytes          = 16;
-    static constexpr bool        is_bit_logical = true;
+    static constexpr std::size_t bits                     = 128;
+    static constexpr std::size_t bytes                    = 16;
+    static constexpr bool        regular_logical_register = true;
```

Comment:
Any awesome bikeshed ideas? `is_wide_logical` for example?

Reply:
I like that. Will amend.